package.json

```diff
@@ -2,21 +2,20 @@
   "name": "tshet-uinh",
   "version": "0.16.0-beta.8",
   "description": "A JavaScript library for the Qieyun phonological system",
-  "main": "dist/tshet-uinh.js",
-  "module": "dist/tshet-uinh.esm.js",
-  "types": "dist/tshet-uinh.d.ts",
+  "type": "module",
+  "main": "./dist/tshet-uinh.cjs",
+  "module": "./dist/tshet-uinh.js",
   "exports": {
     ".": {
-      "types": "./dist/tshet-uinh.d.ts",
-      "import": "./dist/tshet-uinh.esm.js",
-      "default": "./dist/tshet-uinh.js"
+      "import": "./dist/tshet-uinh.js",
+      "require": "./dist/tshet-uinh.cjs"
     }
   },
   "scripts": {
     "postinstall": "python prepare/main.py",
-    "build": "run-p build:*",
+    "build": "run-s build:*",
     "build:rollup": "rollup -c",
-    "build:types": "npm-dts generate -o dist/tshet-uinh.d.ts",
+    "build:types": "api-extractor run --local && bun -e 'await (async f => Bun.write(f, (await Bun.file(f).text()).replace(/^export (?=declare namespace (廣韻|切韻) \\{$)/gm, \"\")))(\"dist/tshet-uinh.d.ts\")' && rm -rf dist/index.* dist/*/",
```

Comment:
This "fix" script seems long enough to be put in a separate file like `fix-dts.js` or so (?)